src/mockchain/lib/Convex/MockChain.hs

```diff
@@ -463,26 +430,8 @@ instance Monad m => MonadBlockchain (MockchainT m) where
     return (slotNo, npSlotLength, utime)
 
 instance Monad m => MonadMockchain (MockchainT m) where
-  setReward cred coin = MockchainT $ do
```

Comment:
These functions are now implemented as helper functions in Convex.Class and operator on the new `modifyMockChainState` and `askNodeParams` interface functions.